### DISCUSSION TITLE
Headless Server patches for disabling the 50% achievement time limit

### DISCUSSION THREAD

**Discussion by wereii** [UnlegitSenpaii/FAE_Linux#14] [Ideas]
I will have a look around in the 2.0.13 binary but my reversing skills are really rusty. 

This string seems to be a promising entry "gui-achievements.not-in-game-long-enough" (though considering it references the GUI locale strings it seems the client might actually be whats handling the condition).

**Comment by UnlegitSenpaii** [comment]
can you explain a little? I haven't played multiplayer let alone while hosting a server. 
Is the progress of achievements halfed when in multiplayer? 
afaik the achievements are handled on the client

**Reply by wereii** [reply]
So, when playing on server, you need to have at least 50% playtime of the save otherwise it won't give you achievements.
I have no idea if this is handled at the client side or server side (client side might actually make more sense here  - though my idea for now has been that either the client or server has to track how much time every player has spent online in that server, with that in mind I assumed this would be tracked over at the server side in the save file).

**Reply by UnlegitSenpaii** [reply]
is there an indicator for the reduced progress? 
since i have found "gui-achievements.not-in-game-long-enough" which might have something to do with this, 
bruh, i didnt read your edit .-.

**Reply by wereii** [reply]
Yes, that's in the achievements menu (top right bar, I think it's a book icon or something like that).

Edit: I don't have this problem personally but connecting to any open, long running vanilla server should show this.
